test/NuGet.Tests.Apex/NuGet.Tests.Apex/NuGetEndToEndTests/NuGetUITestCase.cs

```diff
@@ -60,7 +60,7 @@ public void InstallPackageFromUI()
             VisualStudio.ClearOutputWindow();
             solutionService.SaveAll();
 
-            // Act
```

Comment:
Comments such as 

```cs
// Arrange
// Act
// Assert
```

are commonly used to separate the sections of test. 

They usually don't try to describe the full action.

Reply:
See: https://github.com/NuGet/NuGet.Client/search?q=%2F%2F+act You'll see this pattern a lot.